permission_handler_html/pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: permission_handler_html
 description: Permission plugin for Flutter. This plugin provides the web API to request and check permissions.
-version: 0.1.3+1
+version: 0.1.3+2
 
 homepage: https://github.com/baseflow/flutter-permission-handler
 
 environment:
-  sdk: ">=3.0.5 <4.0.0"
```

Comment:
Is there a reason why you changed the min environment sdk version?

Reply:
Because extension types require dart 3.3.0 . And this library depends on web 0.5 anyway, which already had that retirement: https://github.com/dart-lang/web/blob/v0.5.0/pubspec.yaml
So I don't think this is a breaking change